uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -90,7 +125,14 @@ define([
          * @return {Promise} Resolved with {Object} Updated Leave request
          */
         update: function () {
-          return LeaveRequestAPI.update(this.toAPI());
+          var comments = this.comments;
```

Comment:
If comments are not api data then it will be removed by toAPI call, else it can be removed in toAPI call instead of update? This applies to create call below as well